Podcast/ExternalProfileViewController.swift

```diff
@@ -44,6 +47,8 @@ class ExternalProfileViewController: ViewController, UITableViewDataSource, UITa
         super.viewDidLoad()
         view.backgroundColor = .paleGrey
         
+        if UIScreen.main.nativeBounds.height == 2436 { scrollYOffset -= iPhoneXOffset }
```

Comment:
Let's make `2436` a constant in either `System` of `Constants` .. put in the static variable `iPhoneXHeight` and use that